Algorithm/QuantConnect.Algorithm.csproj

```diff
@@ -123,17 +123,46 @@
     <Compile Include="..\Common\Properties\SharedAssemblyInfo.cs">
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
+    <Compile Include="Alphas\AlphaModel.cs" />
+    <Compile Include="Alphas\AlphaModelExtensions.cs" />
+    <Compile Include="Alphas\AlphaModelPythonWrapper.cs" />
+    <Compile Include="Alphas\CompositeAlphaModel.cs" />
+    <Compile Include="Alphas\IAlphaModel.cs" />
+    <Compile Include="Alphas\INamedModel.cs" />
+    <Compile Include="Alphas\NullAlphaModel.cs" />
     <Compile Include="CandlestickPatterns.cs" />
+    <Compile Include="Execution\ExecutionModel.cs" />
+    <Compile Include="Execution\ExecutionModelPythonWrapper.cs" />
+    <Compile Include="Execution\IExecutionModel.cs" />
+    <Compile Include="Execution\ImmediateExecutionModel.cs" />
+    <Compile Include="Execution\NullExecutionModel.cs" />
     <Compile Include="IndexUniverseDefinitions.cs" />
     <Compile Include="DollarVolumeUniverseDefinitions.cs" />
+    <Compile Include="INotifiedSecurityChanges.cs" />
+    <Compile Include="Portfolio\IPortfolioConstructionModel.cs" />
+    <Compile Include="Portfolio\IPortfolioOptimizer.cs" />
```

Comment:
I see lots of files added here -- makes me think that the _entire_ framework project should have been deleted, but the csproj was still in this diff.